uint8[16] frequency
---
bool result
